<ui version="4.0" >
 <class>frmFind</class>
 <widget class="QWidget" name="frmFind" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>78</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Find &amp; Replace</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <widget class="QCheckBox" name="matchcase" >
     <property name="text" >
      <string>&amp;Match case</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>11</number>
      </property>
      <item>
       <widget class="QCheckBox" name="insource" >
        <property name="text" >
         <string>S&amp;ource</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="tristate" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="intarget" >
        <property name="text" >
         <string>T&amp;arget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="incomment" >
        <property name="text" >
         <string>&amp;Comment</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEdit_2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="lblReplace" >
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="findNext" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string> &amp;Next</string>
       </property>
       <property name="icon" >
        <iconset>../images/next.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="lblFind" >
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="findPrevious" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>  &amp;Previous  </string>
       </property>
       <property name="icon" >
        <iconset>../images/previous.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="replace" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string> &amp;Replace</string>
       </property>
       <property name="icon" >
        <iconset>../images/replace.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QPushButton" name="replaceAll" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string> Replace &amp;All</string>
       </property>
       <property name="icon" >
        <iconset>../images/replaceAll.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>findNext</tabstop>
  <tabstop>findPrevious</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>replace</tabstop>
  <tabstop>replaceAll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
